_data/students.yml

```diff
@@ -303,6 +303,12 @@
   github: Akharote1
   image: aditya_kharote.jpg
   twitter: Akharote1
+  
+- name: Shahaam Marzook
+  country: SriLanka
+  github: ShahaamMarzook
+  image: Shahaam.jpg
+  twitter: @mohammedshahaam
```

Comment:
Please remove the @ from your twitter username

Reply:
@nikit19 
Thanks for pointing the mistake.Thank you
I have corrected it has you pointed it.
 
With respect,
Shahaam Marzook